src/instant/instant.ts

```diff
@@ -321,30 +337,37 @@ function preValidate(state: InstantFormState): InstantFormState {
       field: receiveField,
       priority: 1,
       token: receiveToken,
+      placement: Position.BOTTOM
     });
   }
   if (spendAmount && !receiveAmount) {
     messages.push({
       kind: MessageKind.orderbookTotalExceeded,
       field: spendField,
       priority: 3,
+      placement: Position.TOP
     });
   }
   if (!spendAmount && receiveAmount) {
     messages.push({
       kind: MessageKind.orderbookTotalExceeded,
       field: receiveField,
       priority: 3,
+      placement: Position.TOP
     });
   }
 
   return {
     ...state,
-    messages,
+    messages: messages.sort(byPriority),
```

Comment:
If sorting here, the relevant part of `offerMake` should be sorted as well.
Those two should have similar protocol.

Reply:
But how come the offerMake should be the same. Those 2 parts are not connected. Also yesterday I've investigated the how the errors are handled in the `offerMake` and IMO it's too complex.

Reply:
Actually in general we display 1 message only so I think we have to put only a single error message. I've chatted with MAciek and he is ok with that.